Improve coherence and source legibility of `guide.md` (#613)
dgarijo/Widoco: doc/metadataGuide/guide.md

# Metadata usage in WIDOCO
There are two ways to make WIDOCO aware of the metadata of your ontology:

* Add metadata annotations in your ontology. This makes ontology maintenance easier when you publish new versions, as metadata is included in the ontology. In our [best practice document](https://w3id.org/widoco/bestPractices) we show which properties you should add in your ontology for this purpose. This is the **recommended option**.

> * Declare a `.properties` file: A collection of key-values pair where all the metadata is specified. Since it's a specific file, it has to be maintained separately from the ontology.

Author: ```suggestion
* Declare a `.properties` file: A collection of key-value pairs where all the metadata is specified. Since it's a specific file, it has to be maintained separately from the ontology.
```

|License|[dce:rights], [dcterms:license], [schema:license], [cc:license], [doap:license]|licenseName, licenseURI, licenseIconURL|[Sec 3.7] **[OPTIONAL]**|[Text] or [URI]|[ontology](#onto), [config]|
|Logo            |[foaf:logo], [schema:logo]     |logo                |[Sec 3.8.1] **[OPTIONAL]**   |[URI] |[ontology](#onto), [config]|
|Namespace prefix|[vann:preferredNamespacePrefix]|ontologyPrefix      |[Sec 3.1.2] **[RECOMMENDED]**|[Text]|[ontology](#onto), [config]|
|Namespace URI   |[vann:preferredNamespaceUri]   |ontologyNamespaceURI|[Sec 3.1.1] **[RECOMMENDED]**|[URI] |[ontology](#onto), [config]|
|Modification date|[dcterms:modified], [schema:dateModified] |modified|[Sec 3.4.3] **[OPTIONAL]**   |[Text]|[ontology](#onto), [config]|
|Previous version|[dce:replaces], [dcterms:replaces], [prov:wasRevisionOf], [pav:previousVersion], [owl:priorVersion]|previousVersionURI|[Sec 3.4.1] **[RECOMMENDED]**|[URI]|[ontology](#onto), [config]|
|Publisher|[dcterms:publisher], [dce:publisher], [schema:publisher]|publisher, publisherURI, publisherInstitution, publisherInstitutionURI|[Sec 3.5.3] **[OPTIONAL]**|[Text] or [Organization] or [BNode]|[ontology](#onto), [config]|

> |Similar resources|[rdfs:seeAlso]            |      |[Sec 3.9] **OPTIONAL**    |[Text]                     |[ontology](#onto), [config]|

Author: ```suggestion
|Similar resources|[rdfs:seeAlso]            |      |[Sec 3.9] **[OPTIONAL]**  |[Text]                     |[ontology](#onto), [config]|
```

|Status           |[bibo:status]             |status|[Sec 3.2.4] **[OPTIONAL]**|[Text] or [Status](#status)|[ontology](#onto), [config]|
|Source|[dce:source], [dcterms:source], [prov:hadPrimarySource]|source|[Sec 3.4.5] **[OPTIONAL]**   |[URI] |[ontology](#onto), [config]|
|Title |[dce:title], [dcterms:title], [schema:name]   |ontologyTitle  |[Sec 3.2.1] **[RECOMMENDED]**|[Text]|[ontology](#onto), [config]|
|Version IRI   |[owl:versionIRI]                      |thisVersionURI |[Sec 3.3.1] **[RECOMMENDED]**|[URI] |[ontology](#onto), [config]|

> |Version number|[owl:versionInfo], [schema:schemaVersion]|ontologyRevisionNumber|[Sec 3.3.2] **[RECOMMENDED]**|[Text]|[ontology](#onto), [config]|

Reviewer: Please see comment about links below. I spent quite a lot of time adding them to the table for the very reason that they make my life easier when copy pasting.

|Metadata category |Ontology annotation property                     |Good practices document    |Accepted property value|Example          |
|------------------|-------------------------------------------------|---------------------------|-----------------------|-----------------|
|Definition        |[rdfs:comment], [skos:definition]                |[Sec 4.2] **[RECOMMENDED]**|[Text]                 |[ontology](#term)|
|Deprecation status|[owl:deprecated]                                 |[Sec 4.5.1] **[OPTIONAL]** |[Boolean]              |[ontology](#term)|
|Example           |[vann:example], [skos:example]                   |[Sec 4.4] **[OPTIONAL]**   |[Text]                 |[ontology](#term)|
|Label             |[rdfs:label], [skos:prefLabel], [obo:IAO_0000118]|[Sec 4.1] **[RECOMMENDED]**|[Text]                 |[ontology](#term)|
|Original source   |[rdfs:isDefinedBy], [dce:source]                 |[Sec 4.3] **[OPTIONAL]**   |[URI]                  |[ontology](#term)|
|Rationale         |[vaem:rationale]                                 |[Sec 4.6] **[OPTIONAL]**   |[Text]                 |[ontology](#term)|

> |Status            |[sw:term_status], [obo:IAO_0000114]              |[Sec 4.5.2] **[OPTIONAL]** |[Text]                 |[ontology](#term)|

Reviewer: See comment about links below

```
Widoco will recognize the following properties when describing agents (persons or organizations):

|Metadata category|Ontology annotation property   |Accepted property value            |
|-----------------|-------------------------------|-----------------------------------|

> |Affiliation|[schema:affiliation], [org:memberOf] |[Text] or [Organization] or [BNode]|

Author: ```suggestion
|Affiliation |[schema:affiliation], [org:memberOf]|[Text] or [Organization] or [BNode]|
```

|Family Name      |[schema:familyName], [vcard:family-name], [foaf:family_name]|[Text]                             |
|Full name        |[rdfs:label], [schema:name], [vcard:fn], [foaf:name]        |[Text]                             |
|Given name       |[schema:givenName], [vcard:given-name], [foaf:givenname]    |[Text]                             |
|Email            |[schema:email], [vcard:hasEmail], [foaf:mbox]               |[Text]                             |
|URL              |[schema:url], [vcard:hasURL], [foaf:homepage]               |[URI]                              |

> 

Reviewer: Sorry, why are you removing all the links here? I added them in the first place because having them at the end of the document is a pain to go back and forth. Plus, that way you can copy the metadata property directly from the link instead of having to go to the back to the document all the time. I am fine if you also want to have them at the end of the doc, but I am against removing them here

Author: They are separated just in the source code, to improve legibility of the table. But the [preview/rendering of the .md file](https://github.com/dgarijo/Widoco/blob/ae28862d06600d99dce7a54bcc1588fa4ba3fc93/doc/metadataGuide/guide.md) shows the links correctly at the place they're supposed to be.

Reviewer: Oh, I was not aware of this way of setting up links. Will merge then. Thanks!
